client/src/components/AppointmentCreateForm.js

```diff
@@ -0,0 +1,134 @@
+// @flow
+
+import "react-datetime/css/react-datetime.css";
+
+import React, { useState } from "react";
+import { gql, useMutation } from '@apollo/client';
+
+import Datetime from "react-datetime";
+import { FETCH_APPOINTMENTS } from "./PickupList";
+import { useHistory } from "react-router-dom";
+
+const CREATE_APPOINTMENT = gql`
+  mutation createAppointment($startTime:Int! $endTime:Int! $address:String! $note:String! ){
+    createAppointment(startTime:$startTime, endTime:$endTime, address:$address, note:$note){
+     startTime
+  }}
+`;
+
+const AppointmentCreateForm: () => React$Node = () => {
+
+  const history = useHistory();
+  const backToLanding = () => {
+    history.push("/");
+  };
+
+  const [newFromTime, setNewFromTime] = useState('');
+  const [newToTime, setNewToTime] = useState('');
+  const [newAddress, setNewAddress] = useState('');
+  const [newNote, setNewNote] = useState(' ');
+
+  const [ createAppointment,
+          {loading:mutationLoading, error: mutationError},
+   ] = useMutation(CREATE_APPOINTMENT, {
+    refetchQueries: [{query: FETCH_APPOINTMENTS}],
+    onCompleted: () => backToLanding()
+  });
+
+  const fromTimeInputProps = {
+    placeholder: "Click to open time picker",
+    id: "pickupFromTime",
+    required: "required"
+  };
+  const toTimeInputProps = {
+    placeholder: "Click to open time picker",
+    id: "pickupToTime",
+    required: "required"
+  };
+  
+  var startTime = () => newFromTime.format('X');
```

Comment:
There is a type error here: `newFromTime` is assumed to be a string because of the empty string initial value you give to `useState()`, and string does not have the function `format`.

Additionally, your vscode currently does not have flow type check set up either; you need to set it up so that you start fixing Flow type errors.

two steps:

1. Ensure that running `yarn flow` in your `pickup/client/` folder works and currently it should show you have 5 errors (they are the error I mentioned in this comment).
2. Install a VSCode plugin called [Flow Language Support](https://marketplace.visualstudio.com/items?itemName=flowtype.flow-for-vscode).

Afterwards, you should see those errors directly in VSCode.

Reply:
Ok, I just solved this problem, and test it by `yarn FLOW`.  It shows "No errors!" now.